src/llm/llmnoderesources.cpp

```diff
@@ -182,6 +185,16 @@ Status LLMNodeResources::createLLMNodeResources(std::shared_ptr<LLMNodeResources
     return StatusCode::OK;
 }
 
+Status LLMNodeResources::initializeContinuousBatchingPipeline(
```

Comment:
If we can't return anything else than OK this could be void

Reply:
Right, changed to void